src/04-modality-specific-files/05-task-events.md

```diff
@@ -93,29 +114,76 @@ Example:
 sub-control01/
     func/
         sub-control01_task-emoface_events.tsv
+        sub-control01_task-emoface_events.json
 ```
 
+Example of the content of the TSV file:
+
 ```Text
 onset duration  trial_type  identifier  database  response_time
 1.2 0.6 afraid  AF01AFAF  kdef  1.435
 5.6 0.6 angry AM01AFAN  kdef  1.739
 5.6 0.6 sad AF01ANSA  kdef  1.739
 ```
 
-This file should be accompanied by a data dictionary.  
+The `trial_type` and `identifier` columns from the `*_events.tsv` files might be described
+in the accompanying JSON sidecar as follows:
 
-```Text
-sub-control01/
-    func/
-        sub-control01_task-emoface_events.json
+```JSON
+{
+   "trial_type": {
+       "LongName":   "Emotion image type",
+       "Descripton": "Type of emotional face from Karolinska database that is displayed",
+       "Levels": {
+          "afraid": "A face showing fear is displayed",
+          "angry":  "A face showing anger is displayed",
+		  "sad":    "A face showing sadness is displayed"
+        }
+   },
+   "identifier": {
+       "LongName": "Unique identifier from Karolinska (KDEF) database",
+       "Description": "ID from KDEF database used to identify the displayed image"
+   }
+}
 ```
 
-The `trial_type` and `identifier` columns from the `*_events.tsv` files might be described in this
-dictionary as follows. 
-Note that all other columns SHOULD also be described but are omitted for the sake
-of example.
+Note that all other columns SHOULD also be described but are omitted for the
+sake of example.
```

Comment:
"sake of example" is a somewhat odd phrasing. More idiomatic English (as above):

```suggestion
sake of brevity.
```